src/Platform/index.js

```diff
@@ -109,12 +114,32 @@ export default {
       return getOrSet('network', params)
     },
   },
-  Accesibility: {
+  Accessibility: {
     closedCaptions(params) {
       return getOrSet('closedCaptions', params)
     },
     voiceGuidance(params) {
       return getOrSet('voiceGuidance', params)
     },
   },
+  Personalization: {
+    setEntitlements(entitlements) {
+      return getOrSet('entitlements', entitlements)
+    },
+    addToWatchHistory(history) {
```

Comment:
Same here:

- `params` as the argument name
- consider to rename the method to simply `Platform.Personalization.watchHistory()` that functions as a getter / setter method.

Reply:
I have the same concern here. There's no way to get the watch history.

I tried to put verbs in each of these methods because they're not getter/setters, so a verb identifies what they do.